test/cmdlineTests/yul_leading_or_trailing_dot_in_name/input.yul

```diff
@@ -0,0 +1,8 @@
+{
+  function f() -> u,Q {
+    let j, r._ := f()
+  }
+  {
+    let d, g := f()
+  }
+}
```

Comment:
It's best to clean up the repro a bit before adding it, as long as it does not interfere with the bug reproduction. Especially the repros from fuzzing are usually quite messy and don't follow our style.

```suggestion
{
    function f() -> u, q {
        let j, r._ := f()
    }
    let d, g := f()
}
```

I'd also remove any bits that are not relevant to reproduction, though this one looks fine in that regard.